vizro-core/src/vizro/models/_navigation/nav_item.py

```diff
@@ -0,0 +1,89 @@
+from __future__ import annotations
+
+import itertools
+from typing import Optional
+
+import dash
+import dash_bootstrap_components as dbc
+from dash import html
+from pydantic import Field, root_validator, validator
+
+from vizro.models import VizroBaseModel
+from vizro.models._models_utils import _log_call
+from vizro.models._navigation._navigation_utils import _validate_pages
+from vizro.models._navigation.accordion import Accordion
+from vizro.models.types import NavPagesType
+
+
+class NavItem(VizroBaseModel):
+    """Icon to be used in Navigation Panel of Dashboard.
+
+    Args:
+        pages (NavPagesType): See [NavPagesType][vizro.models.types.NavPagesType].
+        icon (str): URI (relative or absolute) of the embeddable content. Defaults to "home".
```

Comment:
In the description for the icon we should also mention that they can insert any icon-name from the google material icon library and optimally provide a link there